docs/development.rst

```diff
@@ -10,55 +10,48 @@ Licensing
 pySMT is distributed under the APACHE License (see LICENSE file). By
 submitting a contribution, you automatically accept the conditions
 described in LICENSE. Additionally, we ask you to certify that you have
-the right to submit such contributions. We adopt the "Developer 
-Certificate of Origin" approach as done by the Linux kernel.
+the right to submit such contributions. We adopt the “Developer
+Certificate of Origin” approach as done by the Linux kernel.
 
-```
+``Developer Certificate of Origin Version 1.1``
```

Comment:
There is a special syntax for a multi-line verbatim: 
```
previous text ::
  stuff that goes in verbatim
``` 
(notice the indentation of the block)
If you do it by line, you'll see that every line is separated. Instead, this part is a unique block (not sure if I am explaining myself...). We want something that is more like a snippet of code from the python documentation (e.g.,  [1])

[1] https://docs.python.org/2.7/library/multiprocessing.html?#the-process-class